src/commands/remoteDebug/startRemoteDebug.ts

```diff
@@ -13,16 +13,35 @@ import { WebAppTreeItem } from '../../explorer/WebAppTreeItem';
 import { ext } from '../../extensionVariables';
 import * as remoteDebug from './remoteDebugCommon';
 
-export async function startRemoteDebug(node?: SiteTreeItem): Promise<void> {
+let isRemoteDebugging = false;
+
+export async function startRemoteDebug(actionContext: IActionContext, node?: SiteTreeItem): Promise<void> {
+    if (isRemoteDebugging) {
+        throw new Error('Azure Remote Debugging is already started.');

```

Comment:
How long does it take to start remote debugging? If it's a non-negligible amount of time, I think the message should say something like this:
```
Azure Remote Debugging is currently starting or already started.
```

Reply:
Yes it can take some time to start the debugging session, so this makes sense. I updated the message.